xsamtools/vcf.py

```diff
@@ -35,13 +33,15 @@ def _view(input_filepath: str, output_filepath: str, samples: Sequence[str]):
                         "-O", "z",
                         "-S", tf.name,
                         "--threads", f"{2 * cores_available}",
+                        *args,
```

Comment:
Should this check if `-O` or `-S` aren't already being supplied by the user and thus creating duplicate arguments?